### ISSUE TITLE
can we perform clicks?

### ISSUE THREAD

**Issue by Ahtsham0715** [X-SLAYER/flutter_accessibility_service#10]
can we perform clicks on specific part of screen using this plugin outside of our app?.

**Comment by X-SLAYER** [comment]
We can't for the moment 
[2#issuecomment-1209044622](https://github.com/X-SLAYER/flutter_accessibility_service/issues/2#issuecomment-1209044622)

**Comment by Ahtsham0715** [comment]
can you add this feature too?

**Comment by X-SLAYER** [comment]
> can you add this feature too?

Yes I'm working on it

**Comment by Ahtsham0715** [comment]
Great. When it will be available?

**Comment by X-SLAYER** [comment]
I'm still working on some tests but you could try the new updates of clicks and all are in the `automation` branch

**Comment by X-SLAYER** [comment]
You can check an overview of the new updates here 
[Automate YouTube autoscroll](https://www.linkedin.com/posts/x-slayer_flutterabraccessibilityabrservice-automation-activity-7075792072246476800-gmBp?utm_source=share&utm_medium=member_desktop)

**Comment by Ahtsham0715** [comment]
Okay I'll check. Thank you.

**Comment by Ahtsham0715** [comment]
Brother, I tried this branch but it is not capturing text from the screen properly. main branch text capturing was perfect

**Comment by Ahtsham0715** [comment]
Yes, I log all fields. it is working fine on youtube and giving nodeid's but when I try it on Facebook and other apps fields it always returns nodeid as null and also performActionbyText() method not working. Also in the previous version when I log capturedText it was showing all the text from the screen but in the latest version it is not capturing all text from the screen.

**Comment by Ahtsham0715** [comment]
and click actions are not working too

**Comment by X-SLAYER** [comment]
I will re-check it

**Comment by Ahtsham0715** [comment]
ok

**Comment by Ahtsham0715** [comment]
Brother I tried the latest version but the problem is that it is working fine on youtube and other google apps but when it comes to third party apps like facebook or other it always return nodeId as empty or null.

**Comment by Ahtsham0715** [comment]
I have tried on facebook app but it always show nodeId as null or empty but the text part is working at some points

**Comment by Ahtsham0715** [comment]
brother is there any other way to perform actions in apps that are not built with semantic widget?. Actually am working a auto clicker like app which can perform clicks at certain widgets periodically and it can also read or write text from the widgets
